gluten-substrait/src/main/scala/org/apache/gluten/extension/columnar/PushDownInputFileExpression.scala

```diff
@@ -104,6 +104,11 @@ object PushDownInputFileExpression {
           u.copy(children = newFirstChild +: newOtherChildren)
         case p => p.withNewChildren(p.children.map(child => addMetadataCol(child, replacedExprs)))
       }
+
+    def addFallbackTag(plan: SparkPlan): SparkPlan = {
+      FallbackTags.add(plan, "fallback input file expression")
```

Comment:
Can we rephrase with `Project added by rule PushDownInputFileExpression should not be offloaded by design ` or so? Thanks.

Reply:
I thought about it, this project will eventually be removed or merged, so is it more appropriate to keep the original one?

Reply:
OK. I am fine with the message then.